tests/features/communities/tallinn/permissions.feature

```diff
@@ -0,0 +1,71 @@
+@api
+Feature:
+  - As a user, author of a Tallinn Report node, I want to be able to edit the
+    node that I own but I cannot edit other user's Tallinn Reports.
+  - As a moderator, when editing a Tallinn Report node, I want to be able
+    edit the author of the node.
+
+  Scenario: Test permissions on Tallinn Reports.
+    Given users:
+      | Username  | Roles     |
+      | vasile    |           |
+      | dominique |           |
+      | chef      | moderator |
+      | gheorghe  |           |
+    And collection:
+      | title | Kind of Tallinn |
+      | state | validated       |
+    And the following collection user memberships:
+      | collection      | user      |
+      | Kind of Tallinn | vasile    |
+      | Kind of Tallinn | dominique |
+    And tallinn_report content:
+      | title          | author    | collection      |
+      | Romania Report | gheorghe  | Kind of Tallinn |
+      | France Report  | dominique | Kind of Tallinn |
+
+    # A user outside of Tallinn collection is not able to edit any report.
+    Given I am logged in as gheorghe
+    When I visit the tallinn_report content "Romania Report" edit screen
+    Then I should get a 403 HTTP response
```

Comment:
Is there a specific requirement that the user has to be a member to access this? I don't think we have that restriction on Tallinn and I also don't think that the "reporters" will be joinup common users at all. At least not necessarily.

Reply:
Probably regular users can see the reports. This could be tested (and implemented if case).